### ISSUE TITLE
need to update document about DAP

### ISSUE THREAD

**Issue by endlesscpp** [LunarVim/lunarvim.org#186]
I followed the steps in: https://www.lunarvim.org/languages/go.html#lsp-settings

**Comment by codepete** [comment]
You need to run `DBInstall`. Documentation is outdated.

**Comment by kylo252** [comment]
> You need to run `DBInstall`. Documentation is outdated. Currently they're using `dap-buddy.nvim` which is sparse on documentation as well.

`dap-buddy.nvim` should be superseded by `mason.nvim`

**Comment by codepete** [comment]
Yes, but the command for `dap-buddy.nvim` is `DBInstall` for anyone who is stuck. `mason.nvim` is for LSP and not for DAP.
